haxe/ui/core/Component.hx

```diff
@@ -1045,7 +1053,10 @@ class Component extends ComponentBase implements IComponentBase implements IClon
 
     private var _layoutLocked:Bool = false;
     public function lockLayout(recursive:Bool = false) {
```

Comment:
It should work. Currently the layoutLocket var is checked here: https://github.com/haxeui/haxeui-core/blob/5b733d02fa239486c5d462d3c0a5a1802a38582e/haxe/ui/core/Component.hx 

When you unlock the layout, the layout should validate again.